src/main/java/com/internet/shop/controller/user/AddRoleUserController.java

```diff
@@ -25,6 +30,11 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp)
         Set<Role> roles = new HashSet<>(Set.copyOf(user.getRoles()));
         roles.add(role);
         user.setRoles(roles);
+        try {
+            shoppingCartService.getByUserId(userId);
+        } catch (NoSuchElementException exception) {
```

Comment:
is it good to catch runtime exception?

Reply:
What should I do then?
The logic is: I try to get the shopping cart by id, if it doesn't exist -> creating a new one